gems/sorbet/test/snapshot/partial/unnamed_arguments/src/src.rb

```diff
@@ -0,0 +1,5 @@
+class UnnamedArguments
+  def first_method(_, _)
+    nil
+  end
```

Comment:
Do you want to add another method to this test case in a way that Sorbet can't see the definition statically?

```suggestion
  def first_method(_, _)
    nil
  end
  
  define_method(:second_method) do |_, _|
    nil
  end
```